### PR TITLE
nvme: use NVME_GET_FEATURES_SEL definitions

### PR DESCRIPTION
**Pull request by ikegami-t**
These changes are to replace the hardcoded SEL values.

### CODE REVIEW

File: nvme.h [hunk 1]
```diff
@@ -92,6 +92,8 @@ struct nvme_config {
 		OPT_END()                                                              \
 	}
 
+#define NVME_GET_FEATURES_SEL_CHANGED 8
```
[the remark this answers is not in the document]
Comment (ikegami-t) [reply nvme.h:95]: No the sel changed value 8 is not defined in the spec but only used by the nvme-cli get-features command.

[the remark this answers is not in the document]
Comment (igaw) [reply nvme.h:95]: Hmm, this is a little surprising. In this case the field should never see any other values than the one listed in the spec, no? Or is the kernel (ab)using this somehow? Need to dig into it.

Anyway, I think would rename the define to something else, as it looks like the enum defines this value which is not the case.

[the remark this answers is not in the document]
Comment (ikegami-t) [reply nvme.h:95]: The value was implemented by the commit 30b89e5ef. This value is only used by the nvme-cli get-features command but not used by the kernel. Yes the value definition should be separated from the get features command sel field values. So I will do consider to fix this later. Thank you.

[the remark this answers is not in the document]
Comment (ikegami-t) [reply nvme.h]: Just separated NVME_GET_FEATURES_SEL_CHANGED definition and added the changed option instead. Thank you.

**Comment by igaw** [comment]
Thanks!

**Pull request merged**